modules/rpc/yarpc_test.go

```diff
@@ -22,11 +22,11 @@ package rpc
 
 import (
 	"bytes"
-	"context"
 	"testing"
 
 	"go.uber.org/yarpc/encoding/raw"
 	"go.uber.org/yarpc/transport"
+	"golang.org/x/net/context"
```

Comment:
Why this change? Did you accidentally downgrade yarpc? Newer versions of YARPC take the stdlib context.